container-search/src/main/java/com/yahoo/search/ranking/GlobalPhaseHelper.java

```diff
@@ -0,0 +1,152 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.search.ranking;
+
+import ai.vespa.models.evaluation.FunctionEvaluator;
+import ai.vespa.models.evaluation.Model;
+import com.yahoo.component.annotation.Inject;
+import com.yahoo.search.Query;
+import com.yahoo.search.Result;
+import com.yahoo.search.result.Hit;
+import com.yahoo.search.result.HitGroup;
+import com.yahoo.tensor.Tensor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+import java.util.function.Supplier;
+
+public class GlobalPhaseHelper {
+
+    private static final Logger logger = Logger.getLogger(GlobalPhaseHelper.class.getName());
+    private final RankingExpressionEvaluatorFactory factory;
+    private final Set<String> skipProcessing = new HashSet<>();
```

Comment:
This should ideally be deduced from config

Reply:
yes - we can deduce it and wire it same way as rerank-count